src/components/NavIcons.js

```diff
@@ -11,41 +11,60 @@ export default function NavIcons({ activeFilter }) {
   const history = useHistory()
   return (
     <IconWrapper>
-      <BookIcon
-        data-cy="bookicon"
-        onClick={() => {
-          history.push('/reviews')
-          activeFilter('Book')
-        }}
-      />
-      <AudiobookIcon
-        data-cy="audiobookicon"
-        onClick={() => {
-          history.push('/reviews')
-          activeFilter('Audiobook')
-        }}
-      />
-      <PodcastIcon
-        data-cy="podcasticon"
-        onClick={() => {
-          history.push('/reviews')
-          activeFilter('Podcast')
-        }}
-      />
-      <FilmIcon
-        data-cy="filmicon"
-        onClick={() => {
-          history.push('/reviews')
-          activeFilter('Film')
-        }}
-      />
-      <SeriesIcon
-        data-cy="seriesicon"
-        onClick={() => {
-          history.push('/reviews')
-          activeFilter('Series')
-        }}
-      />
+      <BookIconWrapper>
+        <BookIcon
+          data-cy="bookicon"
+          onClick={() => {
+            history.push('/reviews')
+            activeFilter('Book')
+          }}
+        />
+        <p>Books</p>
+      </BookIconWrapper>
+
+      <AudiobookIconWrapper>
+        <AudiobookIcon
+          data-cy="audiobookicon"
+          onClick={() => {
+            history.push('/reviews')
+            activeFilter('Audiobook')
+          }}
+        />
+        <p>Audiobooks</p>
+      </AudiobookIconWrapper>
+
+      <PodcastIconWrapper>
+        <PodcastIcon
+          data-cy="podcasticon"
```

Comment:
I would suggest using kebab-case to make it easier to read: "podcast-icon"

Reply:
Good idea I will change this for all nav-icons